main.go

```diff
@@ -65,22 +67,66 @@ func (stdio) Network() string { return "stdio" }
 // String implements net.Addr interface.
 func (stdio) String() string { return "stdio" }
 
+var (
+	// Set with `-ldflags="-X 'main.version=<the_tag>'"`
+	version = "dev"
+)
+
+const (
+	flagJsonnetPath = "jsonnet-path"
+	flagVersion     = "version"
+)
+
 func main() {
+	if err := run(os.Args); err != nil {
+		log.Fatalf("execution failed: %s", err)
+	}
+}
+
+func run(args []string) error {
+	app := cli.NewApp()
+	app.Name = "jsonnet-language-server"
+	app.Usage = "Server following the LSP spec for jsonnet"
+	app.Action = launchServer
+
+	app.Flags = []cli.Flag{
+		&cli.StringFlag{
```

Comment:
Can this be `--jpath` and short option `-J`?

It would be nice to have options remain consistent with the `jsonnet` tool.